docs/python_docs/python/tutorials/deploy/inference/image_classification_jetson.md

```diff
@@ -0,0 +1,117 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# Image Classication using pretrained ResNet-50 model on Jetson module
+
+This tutorial shows how to install MXNet v1.6 with Jetson support and use it to deploy a pre-trained MXNet model for image classification on a Jetson module.
```

Comment:
v1.7?

Reply:
Do we want to remove this post for v1.7 or update the version number?

Reply:
I mean updating the version number or referring to a version range (to avoid the need to constantly update the number)

Reply:
how about v1.x ? or v1.6+

Reply:
cc @mseth10

Reply:
This tutorial needs to be updated to use v1.7 wheel, which is currently not built/tested. We can skip this tutorial in v1.7 for now. I will test and add it back later.

Reply:
The tutorial also links the build from source instructions, so it may still be of value prior to the wheel being available?

Reply:
Yes, build from source instructions should still work. In that case, we can re-word the tutorial for general 1.x branch. We can keep the 1.6 wheel as an alternative option for now, that can be later updated to 1.7 wheel.

Reply:
Updated version number in this tutorials to `v1.x`, I will go ahead generating the v1.7 website artifacts. You can still update the website content in 1.7 branch in the future.